ncm-ofed/src/main/pan/components/ofed/schema.pan

```diff
@@ -127,7 +128,50 @@ type component_ofed_openib = {
     true;
 };
 
+type component_ofed_partition_property = {
+    @{Port GUID}
+    'guid' : string with match(SELF, '^(ALL(_(SWITCHES|V?CAS|ROUTERS))?|SELF|0x[0-9a-fA-F]{1,10})$')
+    'membership' ? string with match(SELF, '^(limited|full|both)$')
```

Comment:
probably not. i added it here because it can be configured, but i only sortof understand what it is supposed to do, and we typically don't set it 😄